.codemapignore

```diff
@@ -0,0 +1,13 @@
+# showing just the rules, for pad's codemap
```

Comment:
Did you actually mean to add this file?

Reply:
yes, I've added similar files in other repos to visualize and generate code statistics (like the fact that
we reached recently 1 M LOC, of real code (not generated stuff).